tests/Console/Command/ListSetsCommandTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Console\Command;
+
+use PhpCsFixer\Console\Application;
+use PhpCsFixer\Console\Command\ListSetsCommand;
+use PhpCsFixer\Tests\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @internal
+ *
+ * @covers \PhpCsFixer\Console\Command\ListSetsCommand
+ */
+final class ListSetsCommandTest extends TestCase
+{
+    public function testListWithTxtFormat()
+    {
+        $commandTester = $this->doTestExecute([
+            '--format' => 'txt',
+        ]);
+
+        $resultText = $commandTester->getDisplay();
+        $expectedResultStart = ' 1) @DoctrineAnnotation
+      Rules covering Doctrine annotations';
+        static::assertStringStartsWith($expectedResultStart, $resultText);
+        static::assertSame(0, $commandTester->getStatusCode());
+    }
+
+    public function testListWithJsonFormat()
+    {
+        $commandTester = $this->doTestExecute([
+            '--format' => 'json',
+        ]);
+
+        $resultText = $commandTester->getDisplay();
+
+        static::assertJson($resultText);
```

Comment:
Do we check anywhere that this JSON matches schema? Maybe we could do this using https://github.com/estahn/phpunit-json-assertions.

Reply:
`ListSetsCommandTest` is only checking if the command does not crash here.
the check of the content and schema is executed in tests for reporters, eg here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5635/files#diff-153de19a2bcd8d41652d1edadfdc0df1f971ee09364e219e89a9b2187a0d1addR66